db/migrate/20240802110416_remove_equivalency_details_from_application_qualifications.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveEquivalencyDetailsFromApplicationQualifications < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured { remove_column :application_qualifications, :equivalency_details, :string, if_exists: true }
```

Comment:
Friendly reminder that we'll have to migrate a change with `ignore_columns` before dropping this column 
https://github.com/ankane/strong_migrations?tab=readme-ov-file#removing-a-column